src/Events/Integrations/Plugins/Tickets_Wallet_Plus/Controller.php

```diff
@@ -48,9 +48,9 @@ protected function load(): void {
 	 * @return void
 	 */
 	public function register_actions() {
-		add_action( 'tribe_template_after_include:tickets-wallet-plus/pdf/pass/styles', [ $this, 'add_styles_to_pdf' ], 10, 3 );
-		add_action( 'tribe_template_before_include:tickets-wallet-plus/pdf/pass/body/sidebar', [ $this, 'add_venue_to_pdf' ], 10, 3 );
-		add_action( 'tribe_template_before_include:tickets-wallet-plus/pdf/pass/body/post-title', [ $this, 'add_event_date_to_pdf' ], 10, 3 );
+		add_action( 'tribe_template_after_include:tickets-plus/tickets-wallet-plus/pdf/pass/styles', [ $this, 'add_styles_to_pdf' ], 10, 3 );
+		add_action( 'tribe_template_before_include:tickets-plus/tickets-wallet-plus/pdf/pass/body/sidebar', [ $this, 'add_venue_to_pdf' ], 10, 3 );
+		add_action( 'tribe_template_before_include:tickets-plus/tickets-wallet-plus/pdf/pass/body/post-title', [ $this, 'add_event_date_to_pdf' ], 10, 3 );
```

Comment:
@codingmusician should we be replacing or adding hooks ? 

I am thinking here, but currently having ticket plus does not require ETP. So in these cases, these hooks would not be fired. right ?

Reply:
Good call. Yeah. We need to add them, not replace them. Thanks!